src/index.js

```diff
@@ -21,6 +21,15 @@ const renderTodos = () => {
       `;
   });
 
+  document.querySelectorAll('.check').forEach((box) => {
+    box.addEventListener('change', () => {
+      const task = todos.list[+box.parentNode.parentNode.id - 1];
+      task.completed = box.checked;
+      todos.update(task);
+      renderTodos();
+    });
+  });
+
```

Comment:
- [ ] This works 💯 
- [ ] **[OPTIONAL]**  I saw you have two files namely `todo.js` and `todos.js` and both of them contain exactly the same codes, but you only made use of `todos.js` as you imported it in the `index.js`. Please let's remove the `todo.js` file if you're not making use of it anywhere as I noticed. 😄.